docs/api/api.md

```diff
@@ -135,42 +135,36 @@ type CertificateRequestPolicyAllowed struct {
     CommonName *CertificateRequestPolicyAllowedString `json:"commonName,omitempty"`
 
     // DNSNames defines the X.509 DNS SANs that may be requested for.
-    // Accepts wildcards "*".
     // +optional
     DNSNames *CertificateRequestPolicyAllowedStringSlice `json:"dnsNames,omitempty"`
 
-    // IPAddresses defines the X.509 IP SANs that may be requested
-    // for.
+    // IPAddresses defines the X.509 IP SANs that may be requested for.
     // +optional
     IPAddresses *CertificateRequestPolicyAllowedStringSlice `json:"ipAddresses,omitempty"`
 
     // URIs defines the X.509 URI SANs that may be requested for.
     // +optional
     URIs *CertificateRequestPolicyAllowedStringSlice `json:"uris,omitempty"`
 
-    // EmailAddresses defines the X.509 Email SANs that may be
-    // requested for.
+    // EmailAddresses defines the X.509 Email SANs that may be requested for.
     // +optional
     EmailAddresses *CertificateRequestPolicyAllowedStringSlice `json:"emailAddresses,omitempty"`
 
     // IsCA defines whether it is permissible for a CertificateRequest to have
```

Comment:
I don't really understand why we are using the word `permissible` so often in these docs. Shouldn't we just use "whether it is permitted" or "whether it is allowed"?

Reply:
```suggestion
IsCA defines if a CertificateRequest is allowed to have
```

Reply:
Thanks @jsoref! I have now "washed" the API docs for the "permissible" term. I hope it's an improvement. 😉